app/controllers/orga/conferences_controller.rb

```diff
@@ -38,9 +38,8 @@ def conference
 
     def conference_params
       params[:conference] ? params.require(:conference).permit(
-        :name, :url, :location, :twitter, :tickets, :flights, :accomodation,
-        :'starts_on(1i)', :'starts_on(2i)', :'starts_on(3i)', :round,
-        :'ends_on(1i)', :'ends_on(2i)', :'ends_on(3i)', :lightningtalkslots,
+        :name, :url, :location, :twitter, :tickets, :flights, :accomodation, :starts_on, :ends_on,
+        :lightningtalkslots,
```

Comment:
The tests kept failing with the old params, where the dates were split into components. It looks like the components are not needed. (I think Rails converts them automatically?) All tests pass, also no problems with adding and validating dates locally.